nautilus/gen_kube_devsuite

```diff
@@ -72,16 +73,27 @@ def main():
     my_args = 'apt-get -y update; apt-get -y install git awscli build-essential qtbase5-dev rclone;'
     my_args += ' cd /tmp;'
     my_args += ' git clone https://github.com/pypeit/PypeIt.git; cd PypeIt;'
+    my_args += f' git checkout {pargs.pypeit_branch};' 
 
     # Install PypeIt
-    my_args += f' git checkout {pargs.pypeit_branch};' 
-    my_args += ' git pull --ff-only;'
-    my_args += ' pip install -e ".[dev,specutils]";'
-    # Telluric
-    my_args += ' cd pypeit/data/telluric/atm_grids;'
-    my_args += ' aws --endpoint $ENDPOINT_URL s3 cp s3://pypeit/telluric/atm_grids/TelFit_MaunaKea_3100_26100_R20000.fits /tmp/telluric/TelFit_MaunaKea_3100_26100_R20000.fits --no-progress;'
-    my_args += ' aws --endpoint $ENDPOINT_URL s3 cp s3://pypeit/telluric/atm_grids/TelFit_LasCampanas_3100_26100_R20000.fits /tmp/telluric/TelFit_LasCampanas_3100_26100_R20000.fits --no-progress;'
-    my_args += ' ln -s /tmp/telluric/* .;'
+    if pargs.from_wheel:
+        # Build a PypeIt wheel and install from it
+        my_args += ' pip install setuptools extension_helpers;'
+        my_args += ' python setup.py sdist bdist_wheel;'
+        my_args += ' whl_file=(dist/pypeit*.whl);'
+        my_args += ' pip install ${whl_file}[dev];'
```

Comment:
Ah! One quick comment on this: Do we need `[dev,specutils]` here?

Reply:
Nope, at some point in the past I must've been confused about what [dev] does and I never went back and double checked.

Reply:
I ended up removing the "specutils" from the other part of the if because it was unneeded

Reply:
Thanks for checking!  I had also forgotten that we added `specutils` to the `dev` set.